### PR TITLE
WHgEhi-genesis-validator-data

### PR DESCRIPTION
**Pull request by neulerfzco**

### CODE REVIEW

**Review by raj-shekhar1 — COMMENTED** [review]
Proof looks good, 
```
./ethkey verifypop 0xB695CEe3080A58fa021b5e7d721dD1eC391983E5 enode://84c9a23b75bcd0252e0b361f6962a9f360d38f4fe5206cfb2d907074de877edbb1b810fd9cecf2fa64aa6ec4f7816a7f238650d489eaa82d68e8660769c6763d@51.91.220.174:30303 0x1e83A988b4fdCD21849a2b16f6db234ab1473bab 0xa71653e690e47a3829971c8d738e54fb37f13e102759175eb04d63a6fa6f1f6873e0c77e168892e387861c3b277ee3f6 0x0623cc9df187a5ac1f1a533729243a694d4fe3f815b6e5011337bbc3fedf5e3f6e0e690a1ea43e1e861f1db694ae018fee8d6e5c744c65d75ce488739281d5ee01882869a92cbe08ef5eecd4274a2a577e3b426106556bc554f0b6bf9cf1ab5a71776c9be9a970d6ed86d6bf0ccf005ce25f67867337fb7dd4ff351947fd953b4a009444fa0ee0c0f6b4b567a3fa693b383c3651178cb03d889df90842e23dc8e1c6e4847f9d153b839a5d06ae42cc9c089500cdd409f12e83d1f2496be7afadeccc739ef81bdd928d6f12371bb79390bcd8054d0d9eb5dfb9dfb2d304d87d8f5933
===== POP Validation Results =====
Node Key:       true
Oracle Key:     true
Consensus Key:  true
```
@neulerfzco please add `enode://` in the enode URL.

**Comment by neulerfzco** [comment]
Updated the enode url, please confirm.

**Comment by raj-shekhar1** [comment]
test comment

File: genesis-validator-data/WHgEhi.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+    "genesisValidatorData": [
+      {
+        "enode": "enode://0x84c9a23b75bcd0252e0b361f6962a9f360d38f4fe5206cfb2d907074de877edbb1b810fd9cecf2fa64aa6ec4f7816a7f238650d489eaa82d68e8660769c6763d@51.91.220.174:30303",
```
[the remark this answers is not in the document]
Comment (raj-shekhar1) [reply genesis-validator-data/WHgEhi.json]: Thanks, 
checked again, :+1: 

```
./ethkey verifypop 0xB695CEe3080A58fa021b5e7d721dD1eC391983E5 enode://84c9a23b75bcd0252e0b361f6962a9f360d38f4fe5206cfb2d907074de877edbb1b810fd9cecf2fa64aa6ec4f7816a7f238650d489eaa82d68e8660769c6763d@51.91.220.174:30303 0x1e83A988b4fdCD21849a2b16f6db234ab1473bab 0xa71653e690e47a3829971c8d738e54fb37f13e102759175eb04d63a6fa6f1f6873e0c77e168892e387861c3b277ee3f6 0x0623cc9df187a5ac1f1a533729243a694d4fe3f815b6e5011337bbc3fedf5e3f6e0e690a1ea43e1e861f1db694ae018fee8d6e5c744c65d75ce488739281d5ee01882869a92cbe08ef5eecd4274a2a577e3b426106556bc554f0b6bf9cf1ab5a71776c9be9a970d6ed86d6bf0ccf005ce25f67867337fb7dd4ff351947fd953b4a009444fa0ee0c0f6b4b567a3fa693b383c3651178cb03d889df90842e23dc8e1c6e4847f9d153b839a5d06ae42cc9c089500cdd409f12e83d1f2496be7afadeccc739ef81bdd928d6f12371bb79390bcd8054d0d9eb5dfb9dfb2d304d87d8f5933
===== POP Validation Results =====
Node Key:       true
Oracle Key:     true
Consensus Key:  true
```

**Review by raj-shekhar1 — APPROVED** [review]

**Pull request merged**